src/jpeg-checker.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const fs = require('fs');
+const jpeg = require('jpeg-js');
+
+
+var jpeg = require('jpeg-js');
```

Comment:
Please use ES6 syntax.
```suggestion
let jpeg = require('jpeg-js');
```